api/controllers/orderController.js

```diff
@@ -0,0 +1,40 @@
+const {orderService} = require("../services");
+const {catchAsync} = require("../utils/error");
+
+const order = async (req, res) => {
```

Comment:
```suggestion
const getOrderList = catchAsync(async (req, res) => {
```
1. 명확하게 네이밍해주세요~
2. catchAsync 모듈 활용하여 try catch 문은 지워주세요~